tests/utils/common.go

```diff
@@ -199,20 +200,49 @@ func DisableFeatureGate(c client.Client, featureGate string) (*bool, error) {
 	return &previousValue, nil
 }
 
-func removeString(featureGates []string, featureGate string) []string {
+// SetInsecureRegistry sets CDIConfig InsecureRegistries to mark the registry as allowed to be insecure
+func SetInsecureRegistry(c client.Client, registryURL string) error {
```

Comment:
I think AddInsecureRegistry is a better name, a set implies you are overwriting the current value which you are not.

Reply:
renamed